accounts-db/src/ancient_append_vecs.rs

```diff
@@ -907,6 +919,14 @@ impl<'a> AccountsToStore<'a> {
         self.index_first_item_overflow < self.accounts.len()
     }
 
+    /// return # required bytes for the given selector
+    pub fn get_bytes(&self, selector: StorageSelector) -> usize {
```

Comment:
we need a test for this. We probably have a test where we create this splitter thing.

Reply:
Yes. Added.

Reply:
I added an assert that opposite is 0